Plk.Blazor.DragDrop/Dropzone.razor.cs

```diff
@@ -239,6 +249,12 @@ private string GetClassesForSpacing(int spacerId)
         [Parameter]
         public Action<TItem> DragEnd { get; set; }
 
+        [Parameter]
```

Comment:
Please add a `<summary>` to the newly introduced parameters.

Reply:
@brendenblack can you please fix this? :)